src/Elcodi/Store/PaymentBridgeBundle/Services/PaymentBridge.php

```diff
@@ -172,9 +172,13 @@ public function getExtraData()
                     ->getPurchasable()
                     ->getName();
 
+                /*
+                 * Elcodi stores amounts in integer cents (10USD = 1000)
+                 * but Paypal expects decimal (10.00USD)
+                 */
                 $orderLineArray['amount'] = $orderLine
                     ->getAmount()
-                    ->getAmount();
+                    ->getAmount() / 100;
```

Comment:
In the future, we need to extend `Currency` so we can support "other than 2 decimal currencies". So far, this should do.